docs/fastdds/dds_layer/topic/instances.rst

```diff
@@ -0,0 +1,100 @@
+.. include:: ../../../03-exports/aliases-api.include
+
+.. _dds_layer_topic_instances:
+
+Topics, keys and instances
+==========================
+
+Even though, by definition, a Topic corresponds to a single data type, it is possible that several topic instances may
```

Comment:
Suggestion:

A Topic is linked to a data type and each data sample related to a Topic could be seen as an update in the information described by the data type. But it is possible to add a logical separation and have several instances of a Topic, and a data sample should be an update of the information about a concrete  instance.